src/form/dragDropArea/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  root: {
+    height: '100vh',
```

Comment:
hmm? are we sure of that?

Reply:
right, I deleted the internal styles and now user can put his own classes in props